data/settings/icon-grid-id.json.in

```diff
@@ -3,22 +3,24 @@
     "chromium-browser.desktop",
     "org.gnome.Nautilus.desktop",
     "libreoffice-writer.desktop",
-    "rhythmbox.desktop",
-    "com.endlessm.photos.desktop",
+    "libreoffice-calc.desktop",
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.id.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
     "eos-folder-games.directory",
-    "eos-folder-work.directory",
-    "eos-folder-curiosity.directory",
```

Comment:
I was planning to add the apps when they were available, the plan for indonesia is to add as many of them onto the desktop as will fit before overflowing into a folder and I didn't want to add a folder just for translation and duolingo given that maps doesn't actually exist. Not married to this position but I'd expect an update here before we actually ship 3.2.0 when the apps are available.